### DISCUSSION TITLE
[HW5] Can I change number of parameter of unimplemented function?

### DISCUSSION THREAD

**Discussion by Akais24** [prosyslab-classroom/cs424-program-reasoning#450] [Q&A] [answered]
### Your Name

황재영

### Question

먼저 지난 번과 유사한 질문에 미리 죄송합니다

HW5를 구현하면서, 미구현된 vsa.ml의 함수들을 구현할 때,
각 함수들의 파라미터 개수를 변경해도 괜찮은지 여쭤보고 싶습니다

README.md에는 `All the functions you implement must have the same types as described in the module signatures` 라고 작성되어 있는데,
이번 과제에서는 module signatures가 없는 것처럼 보여서 어떤 부분이 same type이어야 하는지 헷갈리는 것 같습니다

---

First, I apologize for asking a similar question as before.

While implementing HW5 and working on the unimplemented functions in vsa.ml,
I would like to ask if it's okay to change the number of parameters for each function.

The README.md states `All the functions you implement must have the same types as described in the module signatures`,
but in this assignment, module signatures seem to be absent, so I'm confused about what should have the "same type".

**Comment by sujin0529** [comment, marked as answer]
안녕하세요.

[해당 질문](https://github.com/prosyslab-classroom/cs424-program-reasoning/discussions/348)과 관련이 있는 것 같아 링크 첨부합니다.

감사합니다.

---
Hi,

I think this question is very similar [previous question](https://github.com/prosyslab-classroom/cs424-program-reasoning/discussions/348).

Thanks.
